plugins/weather/openweather_

```diff
@@ -37,33 +36,34 @@ then
 		graph_category sensors
 		graph_info This graph show the temperature in ${CITY}
 		temp_avg.label avg
-		temp_avg.cdef $KELVIN_BIAS,-
+		temp_avg.cdef temp_avg,$KELVIN_BIAS,-
 
 		multigraph $plugin_name.temp
 		graph_title Temperature in ${CITY}
 		graph_vlabel Celsius
 		graph_category sensors
 		graph_info This graph show the temperature in ${CITY}
 		temp_avg.label avg
-		temp_avg.cdef $KELVIN_BIAS,-
+		temp_avg.cdef temp_avg,$KELVIN_BIAS,-
```

Comment:
I guess, the original cdef was ok? (`temp_avg.cdef $KELVIN_BIAS,-`)

Reply:
Hi,

I still have an error trying to graph : ``rpn expressions without DEF or CDEF variables are not supported``

I got it solved adding ``temp_avg``.

Setup:
  * Debian 9.4 vanilla
  * munin 2.0.33-1
  * rddtool 1.6.0-1

How could I debug further?
I'll try to reproduce with another set up from scratch.

Cheers and sorry for my delayed answer.

Reply:
After taking at a look at the `cdef` again, I think, your change was always on the right track - at least I cannot think of a good way, how the previous `cdef` was interpretet. Probably the `cdef` parser in munin was a bit more sloppy in an older release?

Anyway: if you are happy with the current state, I will merge it.
What do you think?

Reply:
I'm ok with it.

I am not familiar with github workflow though.
Should I rebase my branch on top of master or is it ok like that ?

Reply:
Your changes still do not collide with the current master - thus they can be applied without rebase.

Thank you!